### PR TITLE
#5804 update the Shabondi ENDPOINT_DEFINITION's documentation

### PR DESCRIPTION
**Pull request by vitojeng**
### Checklist
- [x] [Review Contribution Guidelines](https://github.com/oharastream/ohara/blob/master/CONTRIBUTING.md)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#authors))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #5804)
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-shabondi/src/main/scala/oharastream/ohara/shabondi/ShabondiDefinitions.scala [hunk 1]
```diff
@@ -108,7 +108,7 @@ object ShabondiDefinitions {
     .orderInGroup(orderInGroup())
     .optional(Type.STRING)
     .displayName("Endpoint")
-    .documentation("the endpoint of this Shabondi service")
+    .documentation("The endpoint of this Shabondi service. After the service is started, the endpoint will be shown.")
```
Comment (chia7712) [inline ohara-shabondi/src/main/scala/oharastream/ohara/shabondi/ShabondiDefinitions.scala:111]: Is it worth adding url form?

Comment (vitojeng) [reply ohara-shabondi/src/main/scala/oharastream/ohara/shabondi/ShabondiDefinitions.scala:111]: Pardon me, cannot understand what you mean. 
Can you describe more?

Comment (chia7712) [reply ohara-shabondi/src/main/scala/oharastream/ohara/shabondi/ShabondiDefinitions.scala:111]: for example: http://${hostname}:${clientPort}/group=${group}

Comment (vitojeng) [reply ohara-shabondi/src/main/scala/oharastream/ohara/shabondi/ShabondiDefinitions.scala:111]: The `ENDPOINT_DEFINITION` used both in ShabondiSource and ShabondiSink.
I guess it is not suitable to add two url forms in the same line.

Comment (chia7712) [reply ohara-shabondi/src/main/scala/oharastream/ohara/shabondi/ShabondiDefinitions.scala:111]: ok. make sense.

**Review by saivirtue — APPROVED** [review]

**Comment by vitojeng** [comment]
not relate error

**Pull request merged**